src/main/java/org/assertj/core/api/Assertions.java

```diff
@@ -2565,6 +2565,29 @@ public static void useDefaultDateFormatsOnly() {
     return AssertionsForInterfaceTypes.assertThat(actual);
   }
 
+  /**
+   * Creates a new instance of <code>{@link CharSequenceAssert}from a {@link StringBuilder}</code>.
+   *
+   * @param actual the actual value.
+   * @return the created assertion object.
+   * @since 3.11.0
+   */
+  @CheckReturnValue
+  public static AbstractCharSequenceAssert<?, ? extends CharSequence> assertThat(StringBuilder actual) {
+    return new CharSequenceAssert(actual);
+  }
+  /**
+   * Creates a new instance of <code>{@link CharSequenceAssert}from a {@link StringBuffer}</code>.
+   *
+   * @param actual the actual value.
+   * @return the created assertion object.
+   * @since 3.11.0
+   */
+  @CheckReturnValue
+  public static AbstractCharSequenceAssert<?, ? extends CharSequence> assertThat(StringBuffer actual) {
+    return new CharSequenceAssert(actual);
```

Comment:
Should be in `AssertionsForClassTypes.assertThat(actual);` the current code being moved to `AssertionsForClassTypes` (as done for `assertThat(String actual)`)